src/app/pages/news/11.2/07-configEditor.md

```diff
@@ -0,0 +1,17 @@
+## Config File Editor {#configEditor}
+
+Reviewing and editing engine configuration files is simple and powerful with the new built in Config File Editor.
```

Comment:
build in -> build-in

Reply:
I think we are both wrong :)
the word I'd like to use is built-in > https://www.merriam-webster.com/dictionary/built-in